src/chocolatey.tests.integration/infrastructure/cryptography/CryptoHashProviderSpecs.cs

```diff
@@ -23,9 +23,9 @@ namespace chocolatey.tests.integration.infrastructure.cryptography
     using chocolatey.infrastructure.filesystem;
     using Should;
 
-    public class CrytpoHashProviderSpecs
+    public class CryptoHashProviderSpecs

```

Comment:
It's possible that public classes can't be renamed for backwards compatibility reasons, if so, I'm happy to drop.

Reply:
Since this is within the Specs project, this change would be fine.